### PR TITLE
Added check that mobile nav has the same links as desktop nav

### PR DESCRIPTION
**Pull request by jebw**
### Trello card

https://trello.com/c/Z1iYnBtC

### Context

We nearly made a release which had incorrect navigation on the mobile pages

### Changes proposed in this pull request

1. Check that desktop nav includes links for all pages marked as `navigation:`
2. Check that mobile nav includes links for all pages marked as `navigation:`
3. Check that all the links on the desktop nav appear in the mobile nav

### Guidance to review

1. Do the content specs still pass

### CODE REVIEW

File: spec/features/content_pages_spec.rb [hunk 1]
```diff
@@ -87,4 +88,38 @@ def content_urls
       end
     end
   end
+
+  content_urls.slice(0, 1).each do |first_url|
```
Comment (peteryates) [inline spec/features/content_pages_spec.rb:92]: `content_urls.first.tap do |first_url|` could be easier to follow here?

**Review by peteryates — COMMENTED** [review]
Looks good, just one comment about the outer loop. Also 'navigable' :)

File: spec/features/content_pages_spec.rb (hunk 1 above)
Comment (jebw) [reply spec/features/content_pages_spec.rb:92]: :+1:

**Review by peteryates — APPROVED** [review]
Looks good

**Pull request merged**